driver/utils/utils.h

```diff
@@ -75,6 +77,16 @@ template <> constexpr int getObjectHandleType<Connection>() { return SQL_HANDLE_
 template <> constexpr int getObjectHandleType<Descriptor>() { return SQL_HANDLE_DESC; }
 template <> constexpr int getObjectHandleType<Statement>() { return SQL_HANDLE_STMT; }
 
+template <typename T>
+inline constexpr T * ptr_rm_const(const T * ptr) noexcept {
```

Comment:
Why not just `const_cast<T *>(ptr)` everywhere, why adding clutter if you only use if once ?

Reply:
During one of the iterations I used it it many places, but now, yes, these functions are not needed. I'll remove them.